### PR TITLE
Update metainfo

### PR DESCRIPTION
**Pull request by yakushabb**
- Add bugtracker and vcs-browser URLs
- Remove the none oars ratings.

### CODE REVIEW

File: data/metainfo/srain.metainfo.xml.in.in [hunk 1]
```diff
@@ -42,33 +42,14 @@
...
-    <content_attribute id="language-discrimination">none</content_attribute>
-    <content_attribute id="social-chat">intense</content_attribute>
-    <content_attribute id="social-info">none</content_attribute>
-    <content_attribute id="social-audio">none</content_attribute>
-    <content_attribute id="social-location">none</content_attribute>
-    <content_attribute id="social-contacts">none</content_attribute>
-    <content_attribute id="money-purchasing">none</content_attribute>
-    <content_attribute id="money-gambling">none</content_attribute>
-  </content_rating>
+  <content_rating type="oars-1.1">
```
Comment (SilverRainZ) [inline data/metainfo/srain.metainfo.xml.in.in:52]:
```suggestion
  <content_rating type="oars-1.1"/>
```

**Comment by SilverRainZ** [comment]
The metainfo file validate failed on CI (ubuntu-latest, appstream-util 0.7.18-2ubuntu1):

```
/usr/local/share/metainfo/im.srain.Srain.metainfo.xml: FAILED:
• tag-invalid           : <url> type invalid [unknown]
Validation of files failed
```

But it is OK on my laptop (archlinux, appstream-util 0.8.3):

```
$ appstream-util validate-relax im.srain.Srain.metainfo.xml
im.srain.Srain.metainfo.xml: OK
$ appstream-util --version
Version:        0.8.3
```

Does the newly added URL type are introduced in the higher version?

**Comment by yakushabb** [comment]
Appstream-util deprecated. Flathub requires appstreamcli to valiate appdata.

More information: https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#validation

**Review by SilverRainZ — APPROVED** [review]
Thank you~

**Pull request merged**